llvm/include/llvm/MC/MCTargetOptions.h

```diff
@@ -91,8 +91,11 @@ class MCTargetOptions {
   std::string SplitDwarfFile;
   std::string AsSecureLogFile;
 
-  const char *Argv0 = nullptr;
-  ArrayRef<std::string> CommandLineArgs;
+  // This will be set as compiler path in LF_BUILDINFO
```

Comment:
You would need to refer to "debug infos" and "Codeview" in the comment here, since MC is very generic. I would add a single comment, and group the two variables together.